tests/css-epub-text-emphasis/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,80 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>EPUB CSS Text Emphasis</title>
+
+    <style>
+      p.test {
+        @font-face {
+          font-family: "KentenGeneric";
+          src: url(KentenGeneric.otf);
```

Comment:
Ah good point, some fonts apparently may not support the emphasis marks, and I didn't want to take a chance on all the tests failing because the font didn't support, so I added one that did. I'll include a note.